lib/ts/recipe/dashboard/api/listTenants.ts

```diff
@@ -56,6 +76,18 @@ export default async function listTenants(
             passwordless: currentTenant.passwordless,
             thirdParty: currentTenant.thirdParty,
         };
+
+        if (passwordlessContactMethod !== undefined) {
```

Comment:
rewrite this logic.. first get the enabled login methods based on the supertokens.init on the backend sdk, and then further filter that with whats enabled for the tenant. This should apply to all tenants and not just the public tenant

Reply:
done